src/content/docs/zh-cn/reference/errors/get-static-paths-required.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: getStaticPaths() function required for dynamic routes.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+
+> **GetStaticPathsRequired**： 对于动态路由来说，使用 `getStaticPaths()` 函数是必要的。请你确保从动态路由中 `export` 了一个 `getStaticPaths` 函数。(E03015)
+
+## 哪里出了问题?
+在[静态模式](/en/core-concepts/routing/#static-ssg-mode)下，所有路由都必须在构建时确定。因此，动态路由必须`export`一个 `getStaticPaths` 函数，来返回将要生成的不同路径。
```

Comment:
The link should be `zh-cn`. I don't know why check-link ci not working.

Maybe you need to update the `routing` page.